Elements/src/extension.dib

```diff
@@ -0,0 +1,142 @@
+#!markdown
+
+This script is packaged with the NuGet package, and only
+called when the package is imported in a dotnet interactive
+notebook via:
+
+`#r "nuget: Hypar.Elements"`
+
+#!csharp
+
+using System.CommandLine;
+using System.CommandLine.Invocation;
+using Microsoft.DotNet.Interactive;
+using Microsoft.DotNet.Interactive.Formatting;
+using Elements;
+using Elements.Geometry;
+using Elements.Serialization.glTF;
+
+var viewerSrc = @"
+<div id=""main_DIV_ID"" style=""height:400px;width:400px;""></div>
+</div>
+<script type=""module"">
+import * as THREE from 'https://unpkg.com/three@0.126.0/build/three.module.js';
+import { GLTFLoader } from 'https://unpkg.com/three@0.126.0/examples/jsm/loaders/GLTFLoader.js';
+import { OrbitControls } from 'https://unpkg.com/three@0.126.0/examples/jsm/controls/OrbitControls.js';
+const loader = new GLTFLoader();
+const scene = new THREE.Scene();
+
+let loaderScene = new THREE.Scene();
+scene.add(loaderScene);
+
+const width = WIDTH_VAR;
+const height = HEIGHT_VAR;
+const camera = new THREE.PerspectiveCamera(75, width / height, 0.1, 1000);
+const light = new THREE.AmbientLight(0xaaaaaa);
+scene.add(light);
+
+const directionalLight = new THREE.DirectionalLight(0xaaaaaa, 0.5);
+scene.add(directionalLight);
+
+
+const renderer = new THREE.WebGLRenderer();
+renderer.setSize(width, height);
+document.getElementById('main_DIV_ID').appendChild(renderer.domElement);
+
+camera.position.z = 5;
+
+const controls = new OrbitControls(camera, renderer.domElement);
+
+function base64ToArrayBuffer (base64) {
```

Comment:
I assume that this integration doesn't have, like wasm, an "unmarshalled" mode where we can push a byte array instead of serializing to a string?

Reply:
Not as far as I know. We're not even utilizing notebook variable sharing — we're just rendering raw HTML, so for now it's probably gotta be a string.